openpype/hosts/max/plugins/publish/collect_tycache_attributes.py

```diff
@@ -0,0 +1,79 @@
+import pyblish.api
+
+from openpype.lib import EnumDef, TextDef
+from openpype.pipeline.publish import OpenPypePyblishPluginMixin
+
+
+class CollectTyCacheData(pyblish.api.InstancePlugin,
+                         OpenPypePyblishPluginMixin):
+    """Collect Review Data for Preview Animation"""
+
+    order = pyblish.api.CollectorOrder + 0.02
+    label = "Collect tyCache attribute Data"
+    hosts = ['max']
+    families = ["tycache"]
+
+    def process(self, instance):
+        all_tyc_attributes_dict = {}
+        attr_values = self.get_attr_values_from_data(instance.data)
+        tycache_boolean_attributes = attr_values.get("all_tyc_attrs")
```

Comment:
Would it work if we:
```py
tycache_boolean_attributes = attr_values.get("all_tyc_attrs", [])
for attrs in tycache_boolean_attributes:
    all_tyc_attributes_dict[attrs] = True
```

So we skip checking `if tycache_boolean_attributes:` ?